src/Stratis.Bitcoin.IntegrationTests/NodeSyncTests.cs

```diff
@@ -60,6 +60,8 @@ public void CanStratisSyncFromCore()
 
                 //Now check if Core connect to stratis
                 stratisNode.CreateRPCClient().RemoveNode(coreNode.Endpoint);
+                TestHelper.WaitLoop(() => coreNode.CreateRPCClient().GetPeersInfo().Length == 0);
```

Comment:
Why not use `TestHelper.IsNodeConnected(coreNode)`?

Reply:
Yeah I tried then realized this is a bitcoin core node.

Reply:
ah right, fair enough 😆